packages/cdktf/lib/terraform-module.ts

```diff
@@ -33,21 +34,27 @@ export abstract class TerraformModule
   private _providers?: (TerraformProvider | TerraformModuleProvider)[];
   public dependsOn?: string[];
   public forEach?: ITerraformIterator;
+  public readonly createAssetsFromLocalModules?: boolean;
 
   constructor(scope: Construct, id: string, options: TerraformModuleOptions) {
     super(scope, id, "module");
 
-    if (options.source.startsWith("./") || options.source.startsWith("../")) {
-      // Create an asset for the local module for better TFC support
-      const asset = new TerraformAsset(scope, `local-module-${id}`, {
-        path: options.source,
-      });
-
-      // Despite being a relative path already, further indicate it as such for Terraform handling
-      this.source = `./${asset.path}`;
-    } else {
-      this.source = options.source;
+    this.source = options.source;
+
+    if (
+      options.createAssetsFromLocalModules === true ||
+      !("createAssetsFromLocalModules" in options)
```

Comment:
nit: I personally always find it counter intuitive if a boolean var is true by default - could we maybe negate the wording (i.e. skipAssetCreationFromLocalModules or similar / better)?